CalibCalorimetry/EcalCorrectionModules/test/stubs/EcalContainmentCorrectionAnalyzer.cc

```diff
@@ -6,45 +6,44 @@
  *
  */
 
-#include <iostream>
-
-#include <FWCore/Framework/interface/EDAnalyzer.h>
+#include <FWCore/Framework/interface/one/EDAnalyzer.h>
 #include <FWCore/Framework/interface/Event.h>
 #include <FWCore/Framework/interface/MakerMacros.h>
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
 
 #include "CondFormats/DataRecord/interface/EcalGlobalShowerContainmentCorrectionsVsEtaRcd.h"
 #include "CondFormats/EcalCorrections/interface/EcalGlobalShowerContainmentCorrectionsVsEta.h"
-#include "FWCore/Framework/interface/ESHandle.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 
 #include "DataFormats/EcalDetId/interface/EBDetId.h"
 
-class EcalContainmentCorrectionAnalyzer : public edm::EDAnalyzer {
+class EcalContainmentCorrectionAnalyzer : public edm::one::EDAnalyzer<> {
```

Comment:
Although it isn't strictly needed, this module looks like it could be `edm::global::EDAnalyzer<>` which is way more efficient.